code/datums/spell.dm

```diff
@@ -349,6 +352,8 @@ GLOBAL_LIST_INIT(spells, typesof(/obj/effect/proc_holder/spell))
 	if(user && user.ckey)
 		if(create_custom_logs)
 			write_custom_logs(targets, user)
+		if(create_only_logs)
+			user.create_log(ATTACK_LOG, "Cast the spell [name]")
```

Comment:
I think I'd rather have you use the `create_custom_logs` proc for this instead. Having 3 log related fields on a spell is a bit too much. Esp since the current use of this doesn't exclude the `create_attack_logs` field.

Reply:
Done